1136 - Cleanup pages (Round Two) (#1498)
infor-design/enterprise: src/components/bubble/readme.md

    slug: example-index
  pages:
  - name: Defaulting a selected bubble
    slug: test-selected
  - name: Set animation speed
    slug: test-animation
---

## Configuration Options

> The bubble chart is a line chart with the isBubble set that adds the bubble. Plus the z dimension in the data. This is based on the [line chart api]( ../line) see see that api for some more details.

Reviewer: ```suggestion
The bubble chart is a line chart with `isBubble` set. This adds the bubble and the z dimension in the data. This is based on the [line chart api]( ../line) see see that api for some more details.
```

$('#line-example').chart({type: 'bubble', dataset: dataset});
```

## Axis Formatting

> - You can customize and round values on the y axis by setting the `formatterString` option. This uses the d3-format syntax which is documented on the [d3-format api page.](https://github.com/d3/d3-format#api-reference). As an example you can use this pattern to round the tooltip values to currency with 2 decimals.

Reviewer: ```suggestion
- You can customize and round values on the y axis by setting the `formatterString` option. This uses the d3-format syntax which is documented on the [d3-format api page.](https://github.com/d3/d3-format#api-reference). As an example you can use this pattern to round the tooltip values to currency and to two decimals.
```

  z: 'Market Share'
}
},
// Use d3 Format - only value will be formatted
valueFormatterString: {
    z: '0.0%'
}
```

## Selection Model

> - You can initially mark a dot on the chart as selected by passing selected: true in the dataset element.

Reviewer: ```suggestion
- You can initially mark a dot on the chart as selected by passing `selected: true` in the dataset element.
```

```javascript
{
    name: 'Mar',
    value: 14,
    selected: true
}
```

> - You can also use the `getSelected` method to get the currently selected line.

Reviewer: ```suggestion
- You can also use the `getSelected` method to get the current selected line.
```

> - You can also use the `setSelected` method to set the current selected line. For this function you can pass in options such as groupIndex, fieldName and fieldValue to find the associated values.

Reviewer: ```suggestion
- You can also use the `setSelected` method to set the current selected line. For this function you can pass in options such as `groupIndex`, `fieldName` and `fieldValue` to find the associated values.
```
